docs/language-guide.rst

```diff
@@ -269,14 +269,13 @@ Illegal symbols are defined as a a single regular expression:
 Pre-defined modes and regular expressions
 -----------------------------------------
 
-Many languages share common modes and regular expressions. Such expressions are defined in core highlight.js code
-at the end under "Common regexps" and "Common modes" titles. Use them when possible.
+Many languages share common modes and regular expressions. Such expressions are defined in `lib/modes.js`. Once compiled, they be found near the end of the core `highlight.js` code under "Common regexps" and "Common modes" titles. Use them when possible.
```

Comment:
Often times the compiler code is entirely unreadable (minified) does it even make sense to send somehere here to reference it?

Reply:
Github link instead maybe?